### DISCUSSION TITLE
can the image be blurred and used as a background on the container above it?

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#606] [Q&A]
![image](https://user-images.githubusercontent.com/52072811/202886697-d569dc83-7088-4495-8d22-9178aa06ee32.png)

**I have a design idea.  but it looks like it will be difficult to do in flet .  an image that is used as a background and overwritten by a container on top is there a solution for the design like that?**

**Comment by ndonkoHenri** [comment]
Have a look at the Stack control. 
https://flet.dev/docs/controls/stack

Some more examples of it in action could be found here: https://github.com/flet-dev/examples/tree/main/python/controls/stack

**Comment by ndonkoHenri** [comment]
You can blur your image, using any tool out there, then use it in Flet.
And then put the blurred image, and your container in a Stack.
